src/app/authentication/sagas.ts

```diff
@@ -4,7 +4,12 @@ import { call, put, takeLatest } from 'redux-saga/effects';
 import { client } from '../../graphql/api';
 import { Logger } from '../utils/logger';
 
-import { LOGIN, REGISTER } from './actions';
+import { LOGIN, LOGOUT, REGISTER } from './actions';
+
+export function* logoutSaga(): SagaIterator {
+    yield call(client.logout);
+    yield put(LOGOUT.COMPLETED());
+}
 
 export function* registerSaga(action: ReturnType<typeof REGISTER.TRIGGER>): SagaIterator {
     yield call(client.register, action.payload);
```

Comment:
After register we should automatically login, so we should return token, email and username from server and set it to redux